### PR TITLE
Feature/rds utils

### PR DESCRIPTION
**Pull request by alichherawalla**
#### Description of changes
This adds support for $util.rds.toJsonString

#### Description of how you validated changes
manual testing and `yarn test`

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by alichherawalla** [comment]
Hi @cjihrig, 

Is this something that you could help with?

File: packages/amplify-appsync-simulator/src/__tests__/velocity/util/rds.test.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { create } from '../../../velocity/util/index';
+import { mockedInputToRdsJsonString, mockedOutputFromRdsJsonString } from './mock-data';
+import { GraphQLResolveInfo } from 'graphql';
+
+const stubInfo = {} as unknown;
+export const mockInfo = stubInfo as GraphQLResolveInfo;
+var util;
```
Comment (cjihrig) [inline packages/amplify-appsync-simulator/src/__tests__/velocity/util/rds.test.ts:7]:
```suggestion
let util;
```

File: packages/amplify-appsync-simulator/src/__tests__/velocity/util/rds.test.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { create } from '../../../velocity/util/index';
+import { mockedInputToRdsJsonString, mockedOutputFromRdsJsonString } from './mock-data';
+import { GraphQLResolveInfo } from 'graphql';
+
+const stubInfo = {} as unknown;
+export const mockInfo = stubInfo as GraphQLResolveInfo;
```
Comment (cjihrig) [inline packages/amplify-appsync-simulator/src/__tests__/velocity/util/rds.test.ts:6]: Why is this exported?

**Review by cjihrig — COMMENTED** [review]
Left a few comments. The code generally looks good to me, but I'd like to test it a bit.

**Comment by alichherawalla** [comment]
> Left a few comments. The code generally looks good to me, but I'd like to test it a bit.

thanks for the super fast turnaround, I have addressed the comments. 

Thank you and I'll wait to hear back from you after your testing

File: packages/amplify-appsync-simulator/src/velocity/util/rds.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export const rds = {
+  toJsonString: rdsObject => {
+    try {
+      rdsObject = JSON.parse(rdsObject);
+      const rdsJson = (rdsObject?.sqlStatementResults || []).map(statement =>
```
Comment (cjihrig) [inline packages/amplify-appsync-simulator/src/velocity/util/rds.ts:5]:
```suggestion
      const rdsJson = (rdsObject?.sqlStatementResults ?? []).map(statement =>
```

File: packages/amplify-appsync-simulator/src/velocity/util/rds.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        (statement?.records || []).map(record => {
```
Comment (cjihrig) [inline packages/amplify-appsync-simulator/src/velocity/util/rds.ts:6]:
```suggestion
        (statement?.records ?? []).map(record => {
```

File: packages/amplify-appsync-simulator/src/velocity/util/rds.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+          const result = {};
+          record.forEach((row, index) => {
+            result[statement.columnMetadata[index].name] = Object.values(row)[0];
```
Comment (cjihrig) [inline packages/amplify-appsync-simulator/src/velocity/util/rds.ts:9]:
```suggestion
            result[statement?.columnMetadata?.[index]?.name] = Object.values(row)?.[0];
```

**Comment by alichherawalla** [comment]
Done, thank you for the suggestions

**Review by cjihrig — APPROVED** [review]
LGTM, thanks!

**Pull request merged**